### PR TITLE
added back-to-top component #134

### PR DESCRIPTION
**Pull request by kkoryaka**

### CODE REVIEW

**Review by chachasikes — APPROVED** [review]

File: components/back-to-top/src/index.scss [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/* Back to top button */
+
+.back-to-top {
```
Comment (aaronhans) [inline components/back-to-top/src/index.scss:3]: All of these elements are inside the custom element right? If so let's make these more specific by wrapping them in a custom element specifier. If styles aren't inside a custom element use a cagov- prefix on the classnames

**Review by aaronhans — CHANGES_REQUESTED** [review]
Nice work @kkoryaka!

File: components/back-to-top/src/index.scss (hunk 1 above)
Comment (kkoryaka) [reply components/back-to-top/src/index.scss:3]: Not this one... this is appended as #page-container child

Comment (aaronhans) [reply components/back-to-top/src/index.scss:3]: OK 

Want to use:

```
.cagov-back-to-top {
```

We've been starting to follow that pattern in other components so there is less risk of overrides when used on someone else's site.

Comment (kkoryaka) [reply components/back-to-top/src/index.scss]: I have put it inside of the custom element and wrapped css too.

**Review by aaronhans — APPROVED** [review]
nice additional specificity

**Pull request merged**